### ISSUE TITLE
No alt text display for videos

### ISSUE THREAD

**Issue by byarielm** [birdsongapps/Flashes#135]
I’ve been thinking about feeds and accessibility recently, especially for videos (they get alt text less). 

I know Bluesky doesn’t have video alt text accessible visibly for mobile users. I checked Flashes and noticed it doesn’t either.

It would be great to have this implemented. It’s always been a gripe of mine on other platforms that entering the alt text for videos is possible but visibility is non-existent.

Example post with video alt text (visible in the PDS): https://bsky.app/profile/xarielml.bsky.social/post/3lgk4d4xeu22w

UI wise my uninspired idea is just put it next to mute and have a (separate from video playback?) pop up for it like images
